pkgdb/build-aux/clang-tidy-to-json

```diff
@@ -29,140 +28,145 @@ OPTIONS
 
 ENVIRONMENT
   JQ                Command used as \`jq' executable.
-";
-
+"
 
 # ---------------------------------------------------------------------------- #
 
 usage() {
-  if [[ "${1:-}" = "-f" ]]; then
-    echo "$_help_msg";
-  else
-    echo "$_usage_msg";
-  fi
+	if [[ "${1:-}" = "-f" ]]; then
+		echo "$_help_msg"
+	else
+		echo "$_usage_msg"
+	fi
 }
 
-
 # ---------------------------------------------------------------------------- #
 
 # @BEGIN_INJECT_UTILS@
-: "${JQ:=jq}";
-
+: "${JQ:=jq}"
 
 # ---------------------------------------------------------------------------- #
 
-unset _TARGET;
+unset _TARGET
 
 while [[ "$#" -gt 0 ]]; do
-  case "$1" in
-    # Split short options such as `-abc' -> `-a -b -c'
-    -[^-]?*)
-      _arg="$1";
-      declare -a _args;
-      _args=();
-      shift;
-      _i=1;
-      while [[ "$_i" -lt "${#_arg}" ]]; do
-        _args+=( "-${_arg:$_i:1}" );
-        _i="$(( _i + 1 ))";
-      done
-      set -- "${_args[@]}" "$@";
-      unset _arg _args _i;
-      continue;
-    ;;
-    --*=*)
-      _arg="$1";
-      shift;
-      set -- "${_arg%%=*}" "${_arg#*=}" "$@";
-      unset _arg;
-      continue;
-    ;;
-    -u|--usage)    usage;    exit 0; ;;
-    -h|--help)     usage -f; exit 0; ;;
-    -v|--version)  echo "$_version"; exit 0; ;;
-    --) shift; break; ;;
-    -?|--*)
-      echo "$_as_me: Unrecognized option: '$1'" >&2;
-      usage -f >&2;
-      exit 1;
-    ;;
-    *)
-      if [[ -z "${_TARGET:-}" ]]; then
-        _TARGET="$1";
-      else
-        echo "$_as_me: Unexpected argument(s) '$*'" >&2;
-        usage -f >&2;
-        exit 1;
-      fi
-    ;;
-  esac
-  shift;
+	case "$1" in
+	# Split short options such as `-abc' -> `-a -b -c'
+	-[^-]?*)
+		_arg="$1"
+		declare -a _args
+		_args=()
+		shift
+		_i=1
+		while [[ "$_i" -lt "${#_arg}" ]]; do
+			_args+=("-${_arg:$_i:1}")
+			_i="$((_i + 1))"
+		done
+		set -- "${_args[@]}" "$@"
+		unset _arg _args _i
+		continue
+		;;
+	--*=*)
+		_arg="$1"
+		shift
+		set -- "${_arg%%=*}" "${_arg#*=}" "$@"
+		unset _arg
+		continue
+		;;
+	-u | --usage)
+		usage
+		exit 0
+		;;
+	-h | --help)
+		usage -f
+		exit 0
+		;;
+	-v | --version)
+		echo "$_version"
+		exit 0
+		;;
+	--)
+		shift
+		break
+		;;
+	-? | --*)
+		echo "$_as_me: Unrecognized option: '$1'" >&2
+		usage -f >&2
+		exit 1
+		;;
+	*)
+		if [[ -z "${_TARGET:-}" ]]; then
+			_TARGET="$1"
```

Comment:
Suggestion: It also looks like the indentation rules need to be inspected in general because this is nuts.

Reply:
also, not a shell file by extension